community/modules/scheduler/cloud-batch-login-node/variables.tf

```diff
@@ -59,23 +63,48 @@ variable "startup_script" {
   default     = null
 }
 
+variable "job_data" {
+  description = "List of jobs and supporting data for each, typically provided via \"use\" from the cloud-batch-job module."
+  type = list(object({
+    template_contents = string,
+    filename          = string,
+    id                = string
+  }))
+  validation {
+    condition     = length(distinct([for job in var.job_data : job.filename])) == length(var.job_data)
+    error_message = "All filenames in var.job_data must be unique."
+  }
+  validation {
+    condition     = length(distinct([for job in var.job_data : job.id])) == length(var.job_data)
+    error_message = "All job IDs in var.job_data must be unique."
+  }
+}
+
 variable "job_template_contents" {
-  description = "The contents of the Google Cloud Batch job template. Typically supplied by a cloud-batch-job module."
+  description = "Deprecated: The contents of the Google Cloud Batch job template. Typically supplied by a cloud-batch-job module."
```

Comment:
nit: On the deprecated variables, I would say _"Deprecated (use `job_data`):"_.

Reply:
Good point, updated.